lib/src/r_tree/leaf_node.dart

```diff
@@ -58,6 +58,6 @@ class LeafNode<E> extends Node<E> {
 
   clearChildren() {
     _items.clear();
-    _minimumBoundingRect = Rectangle(0, 0, 0, 0);
+    _minimumBoundingRect = null;
```

Comment:
Why the change to `null`?

Reply:
I was talking to @danielway-wk about it and he suggested null is a better representation, which is what we used pre-null-safety. I decided to try changing it in a way that didn't ripple nulls all the way out and was surprised that it actually made performance even better. I haven't dug into why yet so I haven't posted the results, but they were significant. Daniel reproduced the same gain on his machine for my sanity.

Reply:
> ...he suggested null is a better representation

To elaborate on this point: I would argue a node without children has no bounding rectangle. `null` seems like a clearer way to indicate that then `Rectangle(0, 0, 0, 0)` in my opinion.

Reply:
> ...he suggested null is a better representation

To elaborate on this point: I would argue a node without children has no bounding rectangle. `null` seems like a clearer way to indicate that then `Rectangle(0, 0, 0, 0)` in my opinion.

Reply:
I figured that was the case. Of course you could always do `Rectangle(-1, -1, -1, -1)` 😉